frontend/packages/pipelines-plugin/src/components/pipelines/resource-overview/TriggerTemplateResourceLink.tsx

```diff
@@ -15,7 +16,8 @@ const TriggerTemplateResourceLink: React.FC<TriggerTemplateResourceLinkProps> =
   namespace,
   model,
 }) => {
-  const title = model.labelPlural;
+  const { t } = useTranslation();
```

Comment:
Breaks tests in `frontend/packages/pipelines-plugin/src/components/pipelines/resource-overview/__tests__/TriggerTemplateResourceLink.spec.tsx` needs mock.

Reply:
thanks @andrewballantyne !